src/lib/y2network/widgets/gateway.rb

```diff
@@ -21,8 +21,12 @@ def label
       end
 
       def help
-        # TODO: original also does not have help
-        ""
+        _(
+          "<p><b>Gateway</b> defines gateway for given route. Special value here is '-' " \
+            "which means do not use gateway for given destination which for non direct " \
+            "connections result in reject of connection. For direct destination route is " \
```

Comment:
an sometime is used when creating so called device routes. Networking is full of exceptions and corner cases which are not interesting for common user and which we may be should try to hide under hood. In my POV this is what yast should help with. We can simply allow user to define rejecting route (or device route) in a some way, but advanced user should know what is he doing and basic user is not interested in this '-' stuff and only complicates things for him.

Reply:
@mchf but we need have at least in help in which format they can pass it. '-' is known only for users familiar with ifroute. It is not general rejecting route keyword, not?